Lib/fontmake/__main__.py

```diff
@@ -40,6 +41,21 @@ def _loadFeatureWriters(parser, specs):
     return feature_writers
 
 
+def _loadFilters(parser, specs):
```

Comment:
this _loadFilters looks exactly like _loadFeatureWriters, maybe parametrize them